### PR TITLE
py-subword-nmt

### PR DESCRIPTION
**Pull request by Sangu-Mbekelu**

### CODE REVIEW

**Comment by Sangu-Mbekelu** [comment]
@spackbot fix style

**Comment by tldahlgren** [comment]
@spackbot maintainers

File: var/spack/repos/builtin/packages/py-subword-nmt/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-subword-nmt/package.py]:
```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and version sha256.

**Comment by tldahlgren** [comment]
Added python package maintainers as reviewers.

File: var/spack/repos/builtin/packages/py-subword-nmt/package.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    """This repository contains preprocessing scripts to segment text into
+    subword units."""
+
+    # pypi only has the whl file.
+    homepage = "https://github.com/joeynmt/joeynmt"
+    url = "https://github.com/rsennrich/subword-nmt/archive/refs/tags/v0.3.7.zip"
+
+    version("0.3.7", sha256="5c3eafe8d85d872a3bbde722b130fb25db19cc7942561936bfae26b6daf51ba0")
+
+    depends_on("python")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-subword-nmt/package.py]: This line is already added by the base class, you can remove it

**Review by adamjstewart — DISMISSED** [review]

**Comment by adamjstewart** [comment]
@spackbot fix style

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**